modules/ROOT/pages/errors/gql-errors/index.adoc

```diff
@@ -1,3 +1,6 @@
+// This is an automatically generated file. Do not edit it directly.
```

Comment:
As the file is auto-generated but checked-in, I suppose it needs to be re-generated when a new GQLSTATUS is added. How does that work? Is it something that needs to be done manually or does the PR-build/build system somehow solve it?